File: in_go/src/pattern_14.go
package src

import "fmt"

func Pattern_14() {
	row := 7
	column := 7
	var symbol int

	for i := 0; i < row; i++ {
		symbol = row - 1 - i
		for j := 0; j < column; j++ {
			if j <= column-1-i {
				fmt.Print(symbol)
				symbol--
			} else {
				fmt.Print(" ")
			}
		}
		fmt.Println()
	}
}

File: in_rust/src/pattern_14.rs
pub fn pattern_14() {
    let row = 7;
    let column = 7;
    let mut symbol;

    for i in 0..row {
        symbol = row - 1 - i;
        for j in 0..column {
            if j <= column - 1 - i {
                print!("{}", symbol);
                symbol -= 1;
            } else {
                print!(" ");
            }
        }
        println!();
    }
}
